Convert PBS script examples to Slurm.

Sumaya112000 opened the pull request.

Sumaya112000 committed:
Convert PBS script examples to Slurm format

gmfricke reviewed (changes requested):
@Graviton28 We should change the name of the file too

Graviton28 committed:
Revise Slurm job submission documentation

Updated the Slurm job submission guide with examples and explanations for creating scripts and submitting jobs. Enhanced clarity on resource requests and added interactive job instructions.

Graviton28 committed:
Merge pull request #207 from UNM-CARC/Graviton28-patch-1-1

Revise Slurm job submission documentation. Merge sample slurm scripts and submitting jobs.

Graviton28 commented:
Ready for review.

Graviton28 committed:
Rename sbatch_scripts2.md to submitting_sbatch_jobs.md

Graviton28 committed:
Update README.md
